docs/documentation/full-text/sorting.mdx

```diff
@@ -27,6 +27,22 @@ ORDER BY rating DESC
 LIMIT 5;
 ```
 
+When sorting by a field and a limit indexed as `fast => true`, ParadeDB will efficiently return the "top N" fields
+sorted by that field. This is true for all numeric fields (including dates) along with text fields that use the
+`normalizer = 'raw'` setting.
+
+If the field is a text field defined as `fast => true, normalizer => 'lowercase'` (ParadeDB's default for "text" fields),
```

Comment:
Did we change the defaults? In dev its fast = false, normalizer = raw

Reply:
If this is wrong, please feel free to correct it here.  This is what I thought, but I've been wrong before!

Reply:
Ok, let me branch off and PR into your branch for docs